app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1> my blog placeholder </h1>
\ No newline at end of file
+<h1>Blog App</h1>
+
+<ul>
+<% @users.each do |user|%>
+<li>
+<div class='user-content'>
+  <%= image_tag user.photo, alt: "#{user.name} profile image", class: 'profile'%>
+ <div class='user-content2'>
+   <%= link_to user,class:'user-content-details' do %>
+   <h2><%= user.name %></h2>
+   <p class='user-post-total'>Number of posts: <%=user.post_counter%> </p>
+   <%end%>
```

Comment:
- [ ] Great work so far! 👏 However, that's how an image is being displayed on the screen with a valid image URL 👇Kindly make sure to fix this wherever an image is used.

![localhost_3000_users_](https://github.com/emmanuelsteven/MyBlogApp/assets/106119105/135a2049-2c72-4702-bd4e-aa6810c351bb)